package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "license": "GPL-3.0",
   "devDependencies": {
-    "@graphprotocol/graph-cli": "0.71.1",
-    "@graphprotocol/graph-ts": "0.35.1",
+    "@graphprotocol/graph-cli": "^0.91.1",
+    "@graphprotocol/graph-ts": "^0.36.0",
```

Comment:
I updated these trying to get the `yarn deploy-local` task to run.

Reply:
This repo is using really old versions of graphprotocol as you can see, so lot of things might break as you upgrade since they often introduce some breaking changes

Reply:
If AssemblyScript doesn't support iterators, it seems like it never would have supported itertators. It'd be weird to remove support for something like that…

In any case, I replaced the iterators with a `for` loop & the AssemblyScript will now compile to WASM. I deployed the code to the Subgraph Studio & everything seems to work, if you'd like use it.

We're about to redeploy the contracts so the spurious entries will stop showing up in [questchains.xyz](https://questchains.xyz).